src/webgl/p5.Shader.js

```diff
@@ -316,9 +316,10 @@ p5.Shader = class {
   }
 
   _setMatrixUniforms() {
+    const modelMatrix = this._renderer.uModelMatrix;
     const viewMatrix = this._renderer._curCamera.cameraMatrix;
     const projectionMatrix = this._renderer.uPMatrix;
-    const modelViewMatrix = this._renderer.uMVMatrix;
+    const modelViewMatrix = (modelMatrix.copy()).mult(viewMatrix);
```

Comment:
I think the problem with the tests right now might have to do with the fact that we're still using `this._renderer.uMVMatrix` below to calculate the normal matrix, but that property is never updated. Can we add a line below this that does the following?
```js
this._renderer.uMVMatrix = modelViewMatrix;
```